api/checks/webhook.go

```diff
@@ -26,11 +27,10 @@ const (
 	checksForAllUsersFeature = "checksAllUsers"
 )
 
-func isKnownAppID(appID int64) bool {
+func isWPTFYIApp(appID int64) bool {
```

Comment:
If this function only checks the wpt.fyi app now, we don't need to have `switch` any more; in fact, we might as well just get rid of the whole function.